WordPress/Classes/ViewRelated/NUX/SigninSelfHostedViewController.swift

```diff
@@ -95,12 +94,23 @@ import WordPressShared
     }
 
 
+    /// Displays the specified text in the status label.
+    ///
+    /// - Parameters:
+    ///     - message: The text to display in the label.
+    ///
+    func configureStatusLabel(message: String) {
+        statusLabel.text = message
+    }
+
+
     /// Configures the appearance and state of the forgot password button.
     ///
     func configureForgotPasswordButton() {
         var status = ""
-        if statusLabel.text != nil {
-            status = statusLabel.text!
+
+        if let text = statusLabel.text {
+            status = text
```

Comment:
You could also do `let status = statusLabel.text?.nonEmptyString()` and check for `status != nil`.

But more importantly, I doesn't feel right to check the text on a view (that is controlled externally) to decide if a button is enabled or not. I think the "forgot password" button being enabled should follow the same logic than the submit button being enabled. 

I'm starting to miss ReactiveCocoa a tiny bit 😬